### PR TITLE
Auth0 Feature

### PR DESCRIPTION
**Pull request by iammiracle**
What this PR does:

* This PR shows the name and picture of a logged in user

### CODE REVIEW

File: client/js/app/App.jsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+            <Route
+              exact
+              path="/"
+              name="landing"
+              component={(Home)}
+            />
+        </Switch>
+    </Router>
+  );
+  export default App;
```
Comment (enyeInc) [inline client/js/app/App.jsx]: revert there changes

File: pages/callback.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+const Callback = () => {
+    return (
+        <div>
+            <p>Nice</p>
+        </div>
+    );
+};
+
+export default Callback;
```
Comment (enyeInc) [inline pages/callback.js]: What is this component for?

File: server/authentication/Auth.js [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
 
 export default class Auth {
     userProfile;
 
+    accessToken;
+
+    idToken;
+
+    expiresAt;
+
```
Comment (enyeInc) [inline server/authentication/Auth.js]: why are you creating global variables here?

Comment (iammiracle) [reply server/authentication/Auth.js]: I now use Auth0 Passport in my app.js. I am not sure I need it anymore

File: pages/callback.js (hunk 2 above)
Comment (iammiracle) [reply pages/callback.js]: I am not using it, so I have dropped it

File: client/js/app/App.jsx (hunk 1 above)
Comment (iammiracle) [reply client/js/app/App.jsx]: Done, missing importing history.

Comment (enyeInc) [reply client/js/app/App.jsx]: `line 18-19` are not properly indented

Comment (iammiracle) [reply client/js/app/App.jsx]: I have done this

**Review by enyeInc — APPROVED** [review]
LGTM

**Comment by codemutatio** [comment]
Looks Goods to Me

**Pull request merged**